Volume/Views/Onboarding/PublicationDetail/PublicationDetail.swift

```diff
@@ -59,27 +59,37 @@ struct PublicationDetail: View {
     private var backgroundImage: some View {
         ZStack {
             GeometryReader { geometry in
+                let scrollOffset = geometry.frame(in: .global).minY
+                let headerOffset = scrollOffset > 0 ? -scrollOffset : 0
+                let headerHeight = scrollOffset > 0 ? geometry.size.height + scrollOffset : geometry.size.height
+                
                 if let url = publication.backgroundImageUrl {
                     WebImage(url: url)
                         .resizable()
                         .grayBackground()
-                        .aspectRatio(contentMode: .fill)
-                        .frame(width: geometry.size.width, height: 140)
+                        .scaledToFill()
+                        .frame(width: geometry.size.width, height: headerHeight)
                         .clipped()
+                        .offset(x: 0, y: headerOffset)
                 } else {
                     Rectangle() // TODO: Custom image
-                        .frame(width: geometry.size.width, height: 140)
+                        .frame(width: geometry.size.width, height: headerHeight)
                         .foregroundColor(.blue)
+                        .offset(x: 0, y: headerOffset)
                 }
             }
+            .frame(height: 140)
```

Comment:
constants should be in a variable

Reply:
There's no precedent for this in SwiftUI, and I don't think it's necessary to factor out constants that are used only once.

Reply:
+1